src/Async.ts

```diff
@@ -199,20 +259,24 @@ const asyncify =
         f(...args)
 /* eslint-enable @typescript-eslint/no-explicit-any */
 
-/** Allows executing an arbitrary side-effect within a pipeline of
- * `Async` functions, e.g., for logging. Passes the inner value
- * through unchanged.
+/**
+ * Execute an arbitrary side-effect within a pipeline of
+ * `Async` functions. Useful for logging and debugging.
+ * Passes the inner value through unchanged. This function
+ * is sometimes called `do` or `tap`.
+ *
+ * @category Utils
  *
- * @param f should not mutate its arguments
+ * @param f Should not mutate its arguments. See {@link map} if you want to map the inner value.
  *
  * @example
  * await pipe(
  *     Async.of(20),
  *     Async.delay(2_000),
  *     Async.tee(console.log), // logs `20` after 2 seconds
- *     Async.map(double), // double receives the un-altered value `20`
+ *     Async.map(double),      // double receives the un-altered value `20`
  *     Async.start
- * ); // yields `40` after 2 seconds
+ * ) // => 40 (after 2 seconds)
```

Comment:
I like `// =>` here, should we use it elsewhere?

Reply:
Probably makes sense to do that. I would welcome an Issue/PR for that 😅